src/styles/blocks/top-bar.scss

```diff
@@ -0,0 +1,33 @@
+.top-bar {
+  padding-block: 28px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  @include onTablet {
+    padding-block: 64px;
+  }
+
+  &__icons {
+    display: flex;
+    gap: 24px;
+
+    @include onTablet {
+      gap: 40px;
+    }
+  }
+
+  &__logo-link {
+    display: flex;
+  }
+
+  &__logo {
+    height: 16px;
+
+    @include hover(transform, scale(1.1));
```

Comment:
It's better to use ':hover' pseudo-class directly instead of using a mixin for it. Use '&:hover' with 'transform: scale(1.1);' inside.